packages/package.json

```diff
@@ -0,0 +1,121 @@
+{
+  "name": "@hyperledger-labs/cactus",
```

Comment:
@takeutak Is there a reason why you use the `@hyperledger-labs` scope instead of `@hyperledger/cactus-...`